### ISSUE TITLE
[A11yMAS][Accessibility][Keyboard][MAS2.4.7][Eclipse][wipro]:Keyboard focus is not visible for pane present in "Azure Login" dialog 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1525]
**Repro Steps:**
 
1. Open and Start Eclipse.
2. Tab to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Interactive" radio button.
4. Tab to and select "Sign in" button.
5.  In" Azure Login" Enter the "email or phone number" in the edit box and tab up to "click here more options button" then press tab and observe the keyboard focus.

**Expected Result:**
Keyboard focus should be visible for pane present in "Azure Login "pane

**Actual Result:**
Keyboard focus is not visible for pane present in Azure Login Pane

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![keyboard focus is not visible for highlighted pane in azure login dialog](https://user-images.githubusercontent.com/22956130/36191642-2230bc62-1184-11e8-8e83-25128073d63e.jpg)

**Comment by Eskibear** [comment]
under further investigation

**Comment by Eskibear** [comment]
the issue is probably because the focus cannot be changed to inner web page via `tab`

**Comment by Eskibear** [comment]
![image](https://user-images.githubusercontent.com/2351748/38788325-a753d544-4165-11e8-8290-d81fad271276.png)

Eclipse IDE's dialog has the same behavior, it cannot navigate to the whole dialog via tab key either.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749970
